### DISCUSSION TITLE
Using Dough Instead of Pasta in a Dumpling Recipe

### DISCUSSION THREAD

**Discussion by jolopapina** [vectorwing/FarmersDelight#452] [Ideas]
I find it as strange as possible that pasta is used in the recipe for dumplings. It would be much more logical to use the dough.

**Comment by vectorwing** [comment]
Indeed. Wheat Dough used to be in an awkward place (still kinda is), but 0.6 and 1.1 tweaked it a bit; now the dough is used for Dumplings instead. 👍
